platform/ios/src/MGLSDKUpdateChecker.mm

```diff
@@ -0,0 +1,37 @@
+#import "MGLSDKUpdateChecker.h"
+#import "NSBundle+MGLAdditions.h"
+
+@implementation MGLSDKUpdateChecker
+
++ (void)checkForUpdates {
+#if TARGET_IPHONE_SIMULATOR
+    // Abort if running in a playground.
+    if ([[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."]) {
```

Comment:
Does this run as part of the Interface Builder designable agent process? You can use `-[NSProcessInfo(MGLAdditions) mgl_isInterfaceBuilderDesignablesAgent]` to check for that condition.